vignettes/userguide.Rmd

```diff
@@ -0,0 +1,677 @@
+---
+title: "Polars - User Guide for R"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Polars - User Guide for R}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options(rmarkdown.html_vignette.check_title = FALSE)
+```
+
+
+````{comment}
+These functions/methods are either missing, broken, or Vincent can't figure out how to use them.
+
+* `Series_shift`
+* `pl$exclude()`
+* `LazyGroupBy$filter()`
+* `LazyGroupBy$collect()`
+* `pl$concat_str`
+* `pl$col("Defense")$mean()$over(c("Type 1", "Type 2"))$alias("avg_defense_by_type_combination"),`
+* `pl$fold`
+* `DataFrame.explode`
+* Custom R function in an agg on groupby
+  
+Requires new Polars version:
+
+* `df$sample()`
+* `df$describe()`
+````
+
+[The Polars User Guide](https://pola-rs.github.io/polars-book/user-guide/) is an introduction to the Polars DataFrame library. Its goal is to introduce you to Polars by going through examples and comparing it to other solutions. Some design choices are introduced here. The guide will also introduce you to optimal usage of Polars.
+
+Currently, the Polars User Guide includes code examples in Python and Rust. The goal of this guide is to add examples in R. The R examples are not complete yet; when they are complete, our goal is to merge the R examples into the main User Guide. If you want to help, please submit a pull request to the [R polars Github repository.](https://github.com/pola-rs/r-polars)
+
+The current page works as a reference document, for side-by-side comparisons with the Python and Rust examples in the main User Guide.
+
+# Introduction
+
+```{r}
+library(polars)
+```
+
+# Getting started
+
+```{r, include = FALSE}
+df = pl$read_csv("https://j.mp/iriscsv")
+```
+
+```{r, eval = FALSE}
```

Comment:
How about use `{.r}` instead of `{r, eval = FALSE}`?

```suggestion
```{.r}
```

Reply:
3e1d29c165a970f52dbe95512a7b80f06df9af33